dev/tools/test/fake_process_manager.dart

```diff
@@ -0,0 +1,377 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io' as io show ProcessSignal;
+import 'dart:io';
```

Comment:
Is this a mistake?

Reply:
No, the roll dev script can't depend on package:process and the flutter tool anymore.

Reply:
I mean importing `dart:io` twice?

Reply:
ahh, fixed